packages/@react-aria/live-announcer/src/LiveAnnouncer.tsx

```diff
@@ -15,21 +15,20 @@ type Assertiveness = 'assertive' | 'polite';
 /* Inspired by https://github.com/AlmeroSteyn/react-aria-live */
 const LIVEREGION_TIMEOUT_DELAY = 7000;
 
-let liveAnnouncer: LiveAnnouncer | null = null;
-
 /**
  * Announces the message using screen reader technology.
  */
 export function announce(
   message: string,
   assertiveness: Assertiveness = 'assertive',
-  timeout = LIVEREGION_TIMEOUT_DELAY
+  timeout = LIVEREGION_TIMEOUT_DELAY,
+  isIds = false // better name? better description?
```

Comment:
added so i could pass ids to live announcer instead of a string message. this allows me to announce anything the same way without needed to extract the textContent or anything else

Reply:
announceNode/Element or something like that? Maybe this could be a `mode` arg or something that takes 'string | id' and defaults to string?

Reply:
id isn't a type unfortunately, it's also just a string